hbase-protocol-shaded/src/main/protobuf/MasterProcedure.proto

```diff
@@ -53,9 +53,10 @@ enum CreateTableState {
   CREATE_TABLE_PRE_OPERATION = 1;
   CREATE_TABLE_WRITE_FS_LAYOUT = 2;
   CREATE_TABLE_ADD_TO_META = 3;
-  CREATE_TABLE_ASSIGN_REGIONS = 4;
-  CREATE_TABLE_UPDATE_DESC_CACHE = 5;
-  CREATE_TABLE_POST_OPERATION = 6;
+  CREATE_TABLE_ASSIGN_PRIMARY_REGIONS = 4;
```

Comment:
This is an incompatible change? If we have a pending CreateTableProcedure when upgrading we will in trouble?

Reply:
So all procedures proto changes are incompatible changes? The design of procedure needs to be very careful......

Reply:
Yes, so usually we do not change the states for a patch release...

Reply:
Ok. Will just fix this by another way: catch the error and retry to avoid abort RS.